loader/source/menu.c

```diff
@@ -817,24 +818,63 @@ static void Menu_GameSelection_Redraw(MenuCtx *ctx)
 			PrintFormat(DEFAULT_SIZE, (ncfg->Config & (NIN_CFG_SLIPPI_FILE_WRITE)) ? GREEN : RED, MENU_POS_X+340+(22*10),
 					gamelist_y, "%-3s", (ncfg->Config & (NIN_CFG_SLIPPI_FILE_WRITE)) ? "ON" : "OFF");
 
-			// Warn the user if they're running low on USB disk space
-			if ((usb_attached == 1) && (ncfg->Config & (NIN_CFG_SLIPPI_FILE_WRITE)))
+			if ((ncfg->UseUSB == 0) && (usb_attached != 1)  && (ncfg->Config & (NIN_CFG_SLIPPI_FILE_WRITE)))

```

Comment:
can i get some pictures of what these states look like?

Reply:
![PXL_20240926_010615707](https://github.com/user-attachments/assets/a2fb6af4-ed73-40cf-be23-e06719821f6e)
![PXL_20240926_010711920](https://github.com/user-attachments/assets/1b12279d-aa0f-4054-af7b-f55cf95a3e11)
![PXL_20240926_012209324](https://github.com/user-attachments/assets/c7851d5e-822f-484d-9635-eab1ed57e714)